doc/webhook-on-hostnetwork.md

```diff
@@ -0,0 +1,178 @@
+## Admission Webhooks over localhost
+* Use `DaemonSet`: The DaemonSet controller can make Pods even when the scheduler has not been started, which can help cluster bootstrap.
+* Use `hostnetwork` for `PodSpec`
+* The pods must be scheduled on to each master node so that core API server can access the webhook.
+* The Admission Webhook generates`localhost` serving certs. 
+* The Admission Webhook binds to `127.0.0.1` to disable external connection.
+* Use `https://localhost` to define the `URL` of the Admission Webhook.
+
+
+Note that we are not using the following:
+* The `DaemonSet` pods are not fronted by any `Service`
+* No API aggregation, the Admission webhook can not be reached via the `kubernetes.default.svc` service. So we don't get the advantages to registering the webhook server as an aggregated API.   
```

Comment:
As an exercise for our future, can you see if it's possible to register as a service.  Imagine the ImagePolicy admission plugin which we want for kube-apiserver and openshift-apiserver.  Doesn't have to be now and we don't want it for this plugin.